portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,184 @@
+@import url('https://fonts.googleapis.com/css2?family=Crimson+Text&display=swap');
+
+html{
+    background-color: rgb(17, 15, 15);
+    color: #fcf4f4;
```

Comment:
In general, there's no reason to style both html and body (except for some rare cases, like setting a base font size) so most people choose to put base styles like in body